packages/ui/src/context/theme/types.ts

```diff
@@ -10,6 +10,11 @@ export enum ColorType {
   ProtanopiaAndDeuteranopia = 'pnd'
 }
 
+export enum ContentStyleType {
+  Inset = 'inset',
+  NoInset = 'noInset'
```

Comment:
if these are the only two options, a better convention would be store a boolean `inset: boolean`.
but if we think might have more styles in the future, let's not call it `NoInset`, let's give it a name like `Default` or `Full`.

Reply:
Default sounds good👌 FIxed